### PR TITLE
Selenium. HW 1

### PR DESCRIPTION
**Pull request by mhalavin**

### CODE REVIEW

File: tests/mariana_petrushanska/Selemium 1/conftest.py
Comment (kudinov-fedor) [inline tests/mariana_petrushanska/Selemium 1/conftest.py:1]: folder name should stick to the same rulles as python variable,  no spaces
only lowercase letters and spaces

File: tests/mariana_petrushanska/Selemium 1/test_selenium_first_steps.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import pytest
+from selenium.common import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+
+@pytest.mark.usefixtures("login")
```
Comment (kudinov-fedor) [inline tests/mariana_petrushanska/Selemium 1/test_selenium_first_steps.py:6]: no need in this mark, it is autouse True

File: tests/conftest.py [hunk 2]
```diff
@@ -1,35 +1 @@
...
-    # os.mkdir(path)
-    yield path
-    # os.system("rm -rf {}".format(path))
-
-
-@pytest.fixture(scope="session")
-def session() -> WebDriver:
-    session = create_session()
-    yield session
-    session.quit()
```
Comment (kudinov-fedor) [inline tests/conftest.py:35]: you should not remove my conftest fixtures

**Review by kudinov-fedor — CHANGES_REQUESTED** [review]
you should not remove conftest fixtures, its not your file

File: tests/conftest.py (hunk 2 above)
Comment (mhalavin) [reply tests/conftest.py:35]: Sorry, it was done by mistake.

**Review by kudinov-fedor — APPROVED** [review]

**Pull request merged**